src/services/VisorInteractor.js

```diff
@@ -1,7 +1,5 @@
-function visorInteractor(textContent) {
+export default function VisorInteractor(textContent) {
```

Comment:
If we gonna consider it as a component,  then move it to component folder.

But I would like to have only process logical refactor here

Reply:
ooh i think i got it, so since these files are for the intelligence of something i don't need to create them like they're a component? just like they were, modules?

Reply:
yep

Reply:
hey i have to make these changes, but i have resolve the conflicts of this branch, if a make another commit have any problem ?

Reply:
ooh i saw it here, it only did one more commit, nothing that could broke everything :p

Reply:
done